packages/ui/src/accounts/types/balances.ts

```diff
@@ -14,7 +14,7 @@ export const WorkerLocks = [
   'Content Directory Worker',
   'Forum Worker',
   'Membership Worker',
-  'Gateway Worker',
+  'Apps Worker',
   'Builders Worker',
   'HR Worker',
   'Marketing Worker',
```

Comment:
Looks like a double structure with `locks.ts`. Does it make sense to merge them into `app/pages/WorkingGroups/WorkingGroups.tsx`? Note line 62:
>     name: 'Gateway',

Reply:
This name is getting mapped by the helper to the Apps, so no worries.